Formula/vulkan-headers.rb

```diff
@@ -0,0 +1,16 @@
+class VulkanHeaders < Formula
+  desc "Vulkan Header files and API registry"
+  homepage "https://github.com/KhronosGroup/Vulkan-Headers"
+  url "https://github.com/KhronosGroup/Vulkan-Headers/archive/v1.1.99.tar.gz"
+  sha256 "37506deaf56b254193e4fa83195add2617e881efcac3d2ff5f0cf49353eb9aa8"
+  depends_on "cmake" => :build
+
+  def install
+    system "cmake", ".", *std_cmake_args
+    system "make", "install"
+  end
+
+  test do
+    assert_predicate "#{include}/vulkan/vulkan.h", :exist?
```

Comment:
We need to compile a test program against the headers, and run it.

Reply:
But we are unable to link it, without actual libs.

Reply:
How about to try tell `cc` to just produce `o` file without linking the binary?